partner_event/__manifest__.py

```diff
@@ -8,7 +8,7 @@
 
 {
     'name': 'Link partner to events',
-    'version': '10.0.1.0.0',
+    'version': '10.0.1.0.1',
```

Comment:
I'm not sure, but I think this should be `10.0.1.1.0`, since this is a new feature.